src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/CommandLineUtility.cs

```diff
@@ -3,9 +3,12 @@
 
 using System;
 using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
 using System.Text;
```

Comment:
Please remove below import since not used: 
using System.Text;
using NuGet.Commands;

Reply:
Thanks! Fixed.